app/actions/templateManagement/pages.ts

```diff
@@ -3,12 +3,13 @@ import { push } from 'connected-react-router';
 import _ from 'lodash';
 import type { PayloadAction, ReduxThunkAction } from '../types';
 import { ActionType } from '../types';
-import type { SimpleWidgetObj } from '../page';
+import type { SimpleWidgetObj, PageDefinition } from '../page';
 import { forEachWidget } from '../page';
 import Internal from '../../utils/Internal';
 import type { ReduxState } from '../../reducers';
-import type { TemplatePageDefinition } from '../../reducers/templatesReducer';
+import type { PutPagesRequestBody } from '../../../backend/routes/Templates.types';
```

Comment:
General, not necessarily in this PR - how about changing related config(s) to allow shorter backend imports?
```suggestion
import type { PutPagesRequestBody } from 'backend/routes/Templates.types';
```
This could be done in composer as well, so maybe a candidate for separate ticket?

Reply:
Agree. Could I as you to create a ticket for that? Then I'll try push it somewhere in the plans.

Reply:
Created RD-6341

Reply:
Thanks!